### PR TITLE
Add Dockerfile and Makefile

### PR DESCRIPTION
**Pull request by jharrington22**
This PR adds a new Dockerfile and a Makefile to support the deploy scripts

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Build the docker image
+.PHONY: docker-build
+docker-build:
+	$(BUILD_CMD) -t ${IMG} build/.
+
+# Push the docker image
+.PHONY: docker-push
+docker-push:
+	$(DOCKER_CMD) push ${IMG}
```
Comment (jmelis) [inline Makefile]: maybe you want a default for `DOCKER_CMD` ?

Comment (jharrington22) [reply Makefile]: @jmelis here? https://github.com/openshift/aws-account-operator/pull/39/files#diff-b67911656ef5d18c4ae36cb6741b7965R5

Comment (jmelis) [reply Makefile]: but that's BUILD_CMD not DOCKER_CMD?

Comment (jharrington22) [reply Makefile]: ha. My eyes need a rest you are a right sorry I'll fix!

Comment (jmelis) [reply Makefile]: ah I see, it's a typo it should be s/DOCKER_CMD/BUILD_CMD/

Comment (jharrington22) [reply Makefile]: @jmelis it should be different since the one is `docker build` and the other is `docker push` I just got mixed up. All fixed now.

**Comment by jharrington22** [comment]
@jmelis I'll merge if this looks good?

**Pull request merged**